src/sdk/propagation/mod.rs

```diff
@@ -0,0 +1,12 @@
+//! OpenTelemetry Propagators
+pub mod aws;
+mod b3;
+mod baggage;
+mod composite;
+pub mod jaeger;
```

Comment:
Why not export the `jaeger` and `aws` propagators like the others?

Reply:
ah yeah had a feature flag at one point but remembered we're not doing that, can fix